app/controllers/cloud_volume_controller.rb

```diff
@@ -565,6 +565,12 @@ def download_summary_pdf
     super
   end
 
+  # todo: check with MIQ team that this is an acceptable workaround.
+  # It prevents reload() from crashing when it runs tree_select and finds no trees
+  def tree_select
+
+  end
+
```

Comment:
I'm not sure if this is the correct way of doing this? 
Basically, after I delete a volume through the new modal it calls reload() which throws an exception because we don't actually have a tree.